src/app/ApplicationTemplate.Client/ApiEndpoints/UserProfile/UserProfileData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace ApplicationTemplate.Client
+{
+	public record UserProfileData
+	{
+		public UserProfileData(string id, string firstName, string lastName, string email)
+		{
+			Id = id;
+			FirstName = firstName;
+			LastName = lastName;
+			Email = email;
+		}
+
+		[JsonConverter(typeof(StringJsonConverter))]
```

Comment:
Why do we need a `StringJsonConverter` hier ?

Reply:
Because the Json actually contains a numeric value (without quotes) so it doesn't qualify as a string, yet a string is what we want here.